### PR TITLE
Jayant batch update

### PR DESCRIPTION
**Pull request by tavisca-jkanathe**

### CODE REVIEW

File: src/app/shared/components/mat-table/mat-table.component.ts [hunk 1]
```diff
@@ -44,7 +53,14 @@ export class MattableComponent implements OnInit {
   }
  
 
+
   selectRow(id: string): void {
     this.rowSelected.next(id);
   }
+
+  editAction(batchId) {
+   // rowSelected;
```
Comment (lalit-tavisca) [inline src/app/shared/components/mat-table/mat-table.component.ts]: add an event emitter here just like rowSelected. Operation will be decided by parent component. Ex. - this.rowSelected.next(id);

File: src/app/modules/training/components/training-listing/training-listing.component.ts [hunk 2]
```diff
@@ -70,5 +73,6 @@ export class TrainingListingComponent implements OnInit {
         return this.trainings[a].name;
```
Comment (lalit-tavisca) [inline src/app/modules/training/components/training-listing/training-listing.component.ts]: Do we need this?

File: src/app/modules/team/components/create-team/create-team.component.ts [hunk 3]
```diff
@@ -48,7 +48,7 @@ export class CreateTeamComponent implements OnInit {
         }
       },
       (error) => {
-        this.snackBar.open(error.errorInfo[0].message);
+        // this.snackBar.open(error.errorInfo[0].message);
```
Comment (lalit-tavisca) [inline src/app/modules/team/components/create-team/create-team.component.ts]: uncomment it.

File: src/app/modules/team/components/team-list/team-list.component.ts [hunk 4]
```diff
@@ -22,7 +22,7 @@ export class TeamListComponent implements OnInit {
         this.teamList = response.data;
     },
     (error) => {
-      error = error.error.errorInfo[0].message;
+      // error = error.error.errorInfo[0].message;
```
Comment (lalit-tavisca) [inline src/app/modules/team/components/team-list/team-list.component.ts]: show snackbar

File: src/app/modules/batch/services/batch.service.ts [hunk 5]
```diff
@@ -13,9 +20,17 @@ export class BatchService {
...
+    if (environment.isMock) {
+      return of(BATCH_DETAILS);
+    }
+    else {
+      return this.http.get(environment.baseURI + "trainings/Batches/"+ id );
+    }
+  }
+  updateBatch(batch: UpdateBatch, trainingId: string): Observable<any>
   {
+    console.log(batch);
```
Comment (lalit-tavisca) [inline src/app/modules/batch/services/batch.service.ts]: remove console logs

File: src/app/shared/components/mat-table/mat-table.component.ts (hunk 1 above)
Comment (tavisca-jkanathe) [reply src/app/shared/components/mat-table/mat-table.component.ts]: done

File: src/app/modules/training/components/training-listing/training-listing.component.ts (hunk 2 above)
Comment (tavisca-jkanathe) [reply src/app/modules/training/components/training-listing/training-listing.component.ts]: yes for training name

**Pull request closed without merging**